Does not build: missing `Fix`.

When trying to build the project, I am seeing this error:

```
Hode/Rslt/Show/Util.hs:45:25: error:
    Not in scope: type constructor or class ‘Fix’
   |
45 |   :: forall att. Int -> Fix (ExprFWith att)
   |                         ^^^
```

— And many of the same further down the road.

Is this a dependency problem?

--- a/cabal.project
+++ b/cabal.project
@@ -1,0 +1,1 @@
+packages: ./hode-ui/hode-ui.cabal ./hode-test/hode-test.cabal ./hode/hode.cabal

--- a/hode/hode.cabal
+++ b/hode/hode.cabal
@@ -101,7 +101,7 @@ library
               , pointedlist
               , pretty-simple
               , process
-              , recursion-schemes
+              , recursion-schemes <= 5.1.3
               , text
                 , regex-compat
               , vector

